overlay/master_server.py

```diff
@@ -240,6 +241,44 @@ def save_gis_join_counts(counts) -> None:
 
     info(f"Saved {filename}")
 
+def aggregate_metrics_by_state(flattened_metrics: list) -> list:  # returns list(ValidationMetric) at State level
```

Comment:
Function that takes all the county metrics and aggregates them to the state level using that prefix tree implementation.